introduction-to-datasets.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: page
+title: Introduction to Datasets
+---
+
+A dataset is a collection of records, similar to a relational database table. Records are similar to table rows, but the columns can contain not only strings or numbers, but also nested data structures such as lists, maps, and other records.
+
+To define a dataset, Kite minimally requires a [URI](#uris) and a [schema](#schemas).
+
+When using the CLI, you define your dataset using the [`create`][create] command. When using the API, you define your dataset using the [Datasets.create][datasets-create] method.
+
+[create]:{{site.baseurl}}/cli-reference.html#create
+[datasets-create]:{{site.baseurl}}/apidocs/org/kitesdk/data/Datasets.html#create(java.net.URI,%20org.kitesdk.data.DatasetDescriptor)
+
+## URIs
+
+Kite identifies datasets by URI. The URI you provide tells Kite how and where to store data. For example, a dataset created with the URI `dataset:hdfs:/user/cloudera/animals/donkeys` is stored in the `/user/cloudera/animals/donkeys` directory in HDFS.
+
+### Names and Namespaces 
+
+URIs also define a name and namespace for your dataset. Kite uses these values when the underlying system has the same concept (for example, Hive). The name and namespace are typically the last two values in a URI. For example, if you create a dataset using the URI `dataset:hive:animals/donkeys`, Kite stores a Hive dataset named _donkeys_ in the _animals_ namespace. If you create a dataset using the URI `dataset:hdfs:/user/cloudera/animals/donkeys`, Kite stores an HDFS dataset named _donkeys_ in the _animals_ namespace.
```

Comment:
I think the "Kite stores a Hive dataset named..." sentence should use Hive's terminology rather than Kite's. So the URI `dataset:hive:animals/donkeys` will be stored in the Hive table _donkeys_ in the _animals_ Hive db.